### ISSUE TITLE
Show video title

### ISSUE THREAD

**Issue by Fxsch** [TeamPiped/lemmy-piped-link-bot#243]
I hope it's not out of scope for this bot but being able to see the video title before clicking the link would be very helpful

**Comment by yarrim** [comment]
Agreed, and if you are grabbing the video title, I assume other details are available, such as runtime, channel, post date.

These are not always relevant, but they could be, and adding a line after the link containing a few details would not bulk up the message by a lot.
